graphrag_sdk/kg.py

```diff
@@ -58,21 +58,21 @@ def __init__(
         if not isinstance(name, str) or name == "":
             raise Exception("name should be a non empty string")
 
-        # connect to database
+        # Connect to database
         self.db = FalkorDB(host=host, port=port, username=username, password=password)
         self.graph = self.db.select_graph(name)
         ontology_graph = self.db.select_graph("{" + name + "}" + "_schema")
 
         # Load / Save ontology to database
         if ontology is None:
-            # load ontology from DB
-            ontology = Ontology.from_graph(ontology_graph)
+            # Load ontology from DB
+            ontology = Ontology.from_schema_graph(ontology_graph)
+            
+            if len(ontology.entities) == 0 and len(ontology.relations) == 0:
```

Comment:
it's impossible to have 0 entities and some relations, the check should only test entities.